### ISSUE TITLE
Unable to login using domain accounts

### ISSUE THREAD

**Issue by begler1** [microsoft/MCW-Azure-Stack#32]
I am familiar with the Cloud Workshop from last year but trying to test it again now. I get to the point where it says wait for 6 hours for setup to complete and to then login as AZURESTACK\AzureStackAdmin with password demo@pass123 but I get an unable to authenticate error after 10+ hours. I am able to login as local administrator AzS-Host1\administrator with password demo@pass123 and Server Manager shows that AD DS and DNS etc have been installed but I cannot access the domain directly. Hyper-V Manager shows no VMs. I have tried the setup for both labs with the same results. Is there something I am missing? A special password for domain users perhaps?
Thanks
Brian D. Egler 
MCSE/MCT 2019

**Comment by begler1** [comment]
Attaching post-config-transcript.txt
[post-config-transcript.txt](https://github.com/microsoft/MCW-Azure-Stack/files/3808458/post-config-transcript.txt)

Shows the following:
Method invocation failed because [System.Management.Automation.PSCustomObject] does not contain a method named 'Add'.
At C:\Packages\Plugins\Microsoft.Compute.CustomScriptExtension\1.10.2\Downloads\0\post-config.ps1:132 char:13
           $AsdkConfigurator.Add("Autorun", "false")             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Also attaching the post-config.ps1 file referenced as a TXT file.
[post-config.ps1.txt](https://github.com/microsoft/MCW-Azure-Stack/files/3808516/post-config.ps1.txt)

Also screenshot of code block that fails in PowerShell ISE:
![Screenshot (4852)](https://user-images.githubusercontent.com/22414394/68201192-32edf900-ffb9-11e9-9ef0-cd4cb4c67023.png)


Looks like the parameters are not coming across. Could it be that my Test Global Admin password Pa55w.rdPa55w.rd has periods in it? Will try with another password.

**Comment by begler1** [comment]
Same error after changing password. Looks like the parameters are not getting to the post-config.ps1 file properly.

**Comment by begler1** [comment]
Attached Install-ASDK-transcript.txt: 
[Install-ASDK-transcript.txt](https://github.com/microsoft/MCW-Azure-Stack/files/3809109/Install-ASDK-transcript.txt)

See error: 
Resolve-DnsName : time.windows.com : DNS record does not exist
At C:\AzureStackOnAzureVM\Install-ASDK.ps1:440 char:22
         $dnsResult = Resolve-DnsName -Name $timeServer -CacheOnly
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Attached Install-ASDK.ps1 as TXT file:
[Install-ASDK.ps1.txt](https://github.com/microsoft/MCW-Azure-Stack/files/3809140/Install-ASDK.ps1.txt)

Screenshot of PowerShell ISE at error line 440:
![Screenshot (4853)](https://user-images.githubusercontent.com/22414394/68211639-d77b3580-ffcf-11e9-8ed2-0dee1f7bad1f.png)

**Comment by DawnmarieDesJardins** [comment]
@begler1  Thanks, I'll have one of our authors take a look and get back to you.

**Comment by begler1** [comment]
Thanks. If you can let me know the combination of parameter values that will work, that would be helpful.

**Comment by paulopsgility** [comment]
@begler1 Which of the Azure Stack labs are you doing (there are actually two in there)? The regular Azure Stack lab or Azure Stack Operations?

**Comment by DawnmarieDesJardins** [comment]
@begler1 
Hello.  Were you able to complete this workshop?  If not, we need more information from you to better find a solution for you.  Thank you!

**Comment by akn9050** [comment]
Hi @DawnmarieDesJardins 

ASDKconfig script download path got expired as Matt did the changes in his script and updated the script on new url. 
 I have Updated configasdk script download path, as Matt have updated the script download path to http://bit.ly/AzSPoC, old path http://bit.ly/configasdk doesn't work anymore.

Run command also got updated.

Here is the Matt's repo url: https://github.com/mattmcspirit/azurestack

I did the changes and submitted pull request, https://github.com/microsoft/MCW-Azure-Stack/pull/33.

Also i have tested these changes previously.

**Comment by akn9050** [comment]
hi @begler1,

Here is the reference to use correct set of parameters : https://github.com/yagmurs/AzureStack-VM-PoC
In this lab we are deploying the template from [this repository](https://github.com/yagmurs/AzureStack-VM-PoC).

**Comment by begler1** [comment]
Dawnmarie,

Thanks for checking in.

 

We were no able to complete the install of the Azure Stack due to the errors that I posted on GitHub. I put the log files and screenshots on the post for you to review.

I instructed the students to keep an eye on GitHub for a solution to the issue. 

 

Thanks

Brian

 

Brian D. Egler MCSE/MCT 2019

 <mailto:brian.egler@globalknowledge.com> brian.egler@globalknowledge.com 

 

Sent via Office365
